packages/vue3-jest/lib/process.js

```diff
@@ -133,23 +134,28 @@ function processStyle(styles, filename, config) {
 
 module.exports = function(src, filename, config) {
   const { descriptor } = parse(src, { filename })
+  const componentNamespace =
+    getVueJestConfig(config)['componentNamespace'] || vueComponentNamespace
```

Comment:
We have added the `componentNamespace` option to the vue-jest config.
This is because we want the transformer (parser) side of custom blocks to be able to handle them flexibly.

In the case of i18n custom blocks, we are using a common library for generic processing for various bundlers such as vue-i18n-loader, rollup-plugin-vue-i18n, and vite-plugin-vue-i18n. For example, it's handled the below:

https://github.com/intlify/cli/blob/4875ec52e0256bf2cf9723dde999b8efa8c043cd/src/generator/json.ts#L91-L103

In the code in the URL above, the component pass to the function, and uses the variable name `Component`.

In order to reuse this logic in vue-jest transformer, I need to change it a bit, but in any case, being able to specify the variable name allows for flexible processing on the transformer side.